src/Avalonia.Diagnostics/Diagnostics/DevTools.cs

```diff
@@ -45,11 +45,11 @@ void PreviewKeyDown(object? sender, KeyEventArgs e)
                 RoutingStrategies.Tunnel);
         }
 
-        public static IDisposable Open(TopLevel root) => 
-            Open(Application.Current,new DevToolsOptions(),root as Window);
+        public static IDisposable Open(Visual? visual) =>
+            Open(Application.Current, new DevToolsOptions(), visual);
 
-        public static IDisposable Open(TopLevel root, DevToolsOptions options) => 
-            Open(Application.Current, options, root as Window);
+        public static IDisposable Open(Visual? visual, DevToolsOptions options) =>
+            Open(Application.Current, options, visual);
```

Comment:
Seems fine to add methods with Visual, but wouldn't user expect some special behavior if they pass a control in the Open method? I.e., by having only TopLevel we make it clear, that this method opens a dev tools on the toplevel. But not sure about visual.

Reply:
Ah, it's to make it easier for mobile templates. Why is it nullable then?